site2/docs/helm-deploy.md

```diff
@@ -346,7 +346,7 @@ In this example, we name our Helm release `pulsar`.
 ```bash
 helm repo add apache https://pulsar.apache.org/charts
 helm repo update
-helm upgrade --install pulsar apache/pulsar \
+helm install pulsar apache/pulsar \
     --timeout 10m \
     --set [your configuration options]
```

Comment:
```suggestion
helm install pulsar apache/pulsar \
    --timeout 10m \
    --set initialize=true \
    --set [your configuration options]
```
since the `install` command is for the first time deployment, so I suggest adding `--set initialize=true` by default.

Reply:
Emmmm, actually there is a note following on the doc to reminder the config of `initialize=true` for the first time.

![image](https://user-images.githubusercontent.com/10498732/124536557-c5a80080-de4a-11eb-8461-295473b92aed.png)

Reply:
yes, but I think the initial idea of the note is, `helm upgrade --install` is suitable for both first time deployment and upgrade deployment, so according to the note, the user needs to set `--set initialize=true` to init the metadata if it is the first time deployment.
With your change, `helm install` is always for the first time deployment, so I suggest adding `--set initialize=true` since it is not an optional config but a must-have one.

Reply:
@freeznet we have documented this note, so we can not include `--set initialize=true` in the code snippet, thought?

Reply:
@Anonymitaet please refer to my previous comment. `helm upgrade --install` can be used at the first time deployment and future upgrade, so we can have a note tells the user if you are on the first time deployment, please remember to add `--set initialize=true` because it is required for the first time deployment; if you already have pulsar charts installed, then `--set initialize=true` is not required.
But `helm install` is only for the first time deployment, and it is required to have `--set initialize=true`. So IMO, if changed from `helm upgrade --install` to `helm install`, then you have to include `--set initialize=true` in the code block.

Reply:
Also, please check PR #9070

Reply:
Sorry I missed your previous comment. Many thanks for your explanations!
Agree with @freeznet, if we use `helm install`, then we need to add the required argument `--set initialize=true`, or else users might get an error.